### DISCUSSION TITLE
How do I get current time and/or current datetime using time create ? thanks

### DISCUSSION THREAD

**Discussion by birraasmile** [time-rs/time#520] [Question] [answered]
How do I get current time and/or current datetime using time create ? thanks

**Comment by jhpratt** [comment, marked as answer]
There are two methods: [`OffsetDateTime::now_utc`](https://time-rs.github.io/api/time/struct.OffsetDateTime.html#method.now_utc) and [`OffsetDateTime::now_local`](https://time-rs.github.io/api/time/struct.OffsetDateTime.html#method.now_local).

**Reply by birraasmile** [reply]
👍 , thanks @jhpratt  ....sorry I had failed to notice that.
